packages/flutter_driver/lib/src/timeline_summary.dart

```diff
@@ -173,8 +163,26 @@ class TimelineSummary {
         .reduce((double a, double b) => math.max(a, b));
   }
 
-  List<TimedEvent> _extractFrameEvents() => _extractCompleteEvents('Frame');
+  double _averageInMillis(List<Duration> durations) {
```

Comment:
I thought about that, but these TimedEvents would be lies.  Their start time would be accurate but their end time wouldn't coorespond to anything that actually happened at that time.

If you don't like the duplication, I can instead write a function that takes a List<TimedEvent> and returns a List<Duration> that can then be passed to these functions to average/max.